model/storage/DeprecatedStorage.php

```diff
@@ -85,15 +86,15 @@ public function load(core_kernel_classes_Resource $test)
     public function deleteContent( core_kernel_classes_Resource $test)
     {
         /** @var \core_kernel_classes_Literal $directoryId */
-        $directoryId = $test->getOnePropertyValue($this->getProperty(TEST_TESTCONTENT_PROP));
+        $directoryId = $test->getOnePropertyValue($this->getProperty(TestService::TEST_TESTCONTENT_PROP));
         if(is_null($directoryId)){
             throw new \common_exception_FileSystemError(__('Unknown test directory'));
         }
 
         $directory = \tao_models_classes_service_FileStorage::singleton()->getDirectoryById($directoryId->literal);
         $directory->deleteSelf();
 
-		$test->removePropertyValues($this->getProperty(TEST_TESTCONTENT_PROP));
+		$test->removePropertyValues($this->getProperty(TestService::TEST_TESTCONTENT_PROP));
```

Comment:
it is deprecated, why do you use it?

Reply:
It's just a replace this entire class it's deprecated. So I am not using it just replace it.

Reply:
what do I mean:
https://github.com/oat-sa/extension-tao-test/blob/476db8e24a01aee6765da5de09da4908ceaeb255/models/classes/class.TestsService.php#L46
it's deprecated, so you can just use
TestService::PROPERTY_TEST_CONTENT

Reply:
I see now, I have replace all of TEST_TESTCONTENT_PROP with  PROPERTY_TEST_CONTENT, Thanks